Core/Lib/Exceptions/ValidationException.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Lens.Core.Lib.Exceptions
+{
+    public class ValidationException : System.ComponentModel.DataAnnotations.ValidationException
+    {
+        public ValidationException(string message) : base(message)
+        {
+
+        }
+
+        public ValidationException() : base()
+        {
+        }
+
+        public ValidationException(System.ComponentModel.DataAnnotations.ValidationResult validationResult, System.ComponentModel.DataAnnotations.ValidationAttribute? validatingAttribute, object? value) : base(validationResult, validatingAttribute, value)
```

Comment:
add using System.ComponentModel.DataAnnotations and remove the repetitive namespace.